config/options.go

```diff
@@ -24,6 +24,8 @@ import (
 	"github.com/uber/jaeger-lib/metrics"
 
 	"github.com/uber/jaeger-client-go"
+
+	otobserver "github.com/opentracing-contrib/go-observer"
```

Comment:
I would prefer not to have an explicit dependency on this package, but instead just re-define its interfaces here. Due to duck typing it should still be compatible, but Jaeger will be isolated from the potential API changes in the contrib package.